app/src/main/java/co/infinum/complexify/MainActivity.kt

```diff
@@ -0,0 +1,57 @@
+package co.infinum.complexify
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import androidx.appcompat.app.AppCompatActivity
+import co.infinum.complexify.databinding.ActivityMainBinding
+import co.infinum.complexify.ui.setComplexityListener
+
+class MainActivity : AppCompatActivity() {
```

Comment:
I left some library implementation comments, so call sites like this will most probably change.